src/main/java/net/revelc/code/formatter/FormatterMojo.java

```diff
@@ -565,7 +565,7 @@ private List<File> addCollectionFiles(final Resource resource) {
         for (final String filename : ds.getIncludedFiles()) {
             foundFiles.add(new File(newBasedir, filename));
         }
-        return foundFiles;
+        return List.of(foundFiles.toArray(new File[] {}));
```

Comment:
This is a lot of converting back and forth. This line, and the previous loop and temporary foundFiles object can be entirely replaced by:

```java
        return Stream.of(ds.getIncludedFiles()).map(filename -> new File(newBasedir, filename))
            .collect(Collectors.toUnmodifiableList());
```

Reply:
agreed, change coming shortly.